package_manager/cloudbuild.yaml

```diff
@@ -0,0 +1,30 @@
+# A cloud build config to release a PAR binary of
+# the dpkg_parser.par used for the package manager bazel rules
+
+# Takes a substitution of _VERSION
```

Comment:
I added a trigger on the project for the master branch and versioned it with $COMMIT_SHA.  Not sure if this is the best solution, since we really don't care if its built on _every_ commit, but tags are probably not appropriate since this is just a tool that lives in the distroless repo.